src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
I'm happy to send a PR that puts the comment back to how it was. We may though consider putting a link to a description of the algorithm in the code, as @kurtschelfthout suggested [here](https://github.com/fscheck/FsCheck/pull/221#issuecomment-211901685).